lib/executor/constant_arrival_rate_test.go

```diff
@@ -227,7 +228,7 @@ func TestConstantArrivalRateRunCorrectTiming(t *testing.T) {
 				// check that we got around the amount of VU iterations as we would expect
 				var currentCount int64
 
-				for i := 0; i < 3; i++ {
+				for i := 0; i < seconds; i++ {
```

Comment:
Erm, I'm confused - won't `seconds` be `0`, so this loop will never execute? :confused: